swagger-codegen-templates/routes.mustache

```diff
@@ -0,0 +1,26 @@
+=begin
+{{> info}}
+=end
+Rails.application.routes.draw do
+
+  mount SwaggerUiEngine::Engine, at: '/api'
+
+  def add_swagger_route http_method, path, opts = {}
+    full_path = path.gsub(/{(.*?)}/, ':\1')
+    constraint = DefaultsConstraint
```

Comment:
@mkanoor @hsong-rh - This is required because we agreed to pluralize the `Constraint` to `AdminsConstraint` and `DefaultsConstraint`. When no tag is present in the `swagger.yaml` file, it defaults to a singular `default` which would result in a constraint lookup of `DefaultConstraint` which doesn't exist.

Reply:
👍